barretenberg/cpp/pil/vm2/opcodes/get_env_var.pil

```diff
@@ -0,0 +1,149 @@
+ * |   2   | transactionFee |         0         |          0          |          0         |         2         |     0      |     0     |        1         |       0       |         0       |      0       |      0         |
+ * |   3   | chainId        |         0         |          1          |          0         | CHAIN_ID_ROW_IDX  |     0      |     0     |        0         |       0       |         0       |      0       |      0         |
+ * |   4   | version        |         0         |          1          |          0         | VERSION_ROW_IDX   |     0      |     0     |        0         |       0       |         0       |      0       |      0         |
+ * |   5   | blockNumber    |         0         |          1          |          0         | BLOCK_NUM_ROW_IDX |     0      |     0     |        0         |       0       |         0       |      0       |      0         |
+ * |   6   | timestamp      |         0         |          1          |          0         | TIMESTAMP_ROW_IDX |     0      |     0     |        0         |       0       |         0       |      0       |      0         |
+ * |   7   | feePerL2Gas    |         0         |          0          |          1         | GAS_FEES_ROW_IDX  |     0      |     0     |        0         |       1       |         0       |      0       |      0         |
+ * |   8   | feePerDaGas    |         0         |          1          |          0         | GAS_FEES_ROW_IDX  |     0      |     0     |        0         |       0       |         0       |      0       |      0         |
+ * |   9   | isStaticCall   |         0         |          0          |          0         |         0         |     0      |     0     |        0         |       0       |         1       |      0       |      0         |
+ * |  10   | l2GasLeft      |         0         |          0          |          0         |         0         |     0      |     0     |        0         |       0       |         0       |      1       |      0         |
+ * |  11   | daGasLeft      |         0         |          0          |          0         |         0         |     0      |     0     |        0         |       0       |         0       |      0       |      1         |
+ * |  12+  | (invalid)      |         1         |          0          |          0         |         0         |     0      |     0     |        0         |       0       |         0       |      0       |      0         |
+ * +-------+----------------+-------------------+---------------------+--------------------+-------------------+------------+-----------+------------------+---------------+-----------------+--------------+----------------+
+ */
+
+namespace execution; // this is a virtual gadget that shares rows with the execution trace
+
+    // No relations will be checked if this identity is satisfied.
+    #[skippable_if]
+    sel = 0; // from execution.pil.
```

Comment:
Can this even be `sel_get_env_var`? The more specific we can be, the better. Otherwise we'll run the checks for every execution row even when it's not a get_env_var opcode.

I think we probably want this to be `sel_get_env_var` even if some changes to the relations are needed.